datafusion/physical-expr/src/aggregate/sum.rs

```diff
@@ -146,7 +139,7 @@ impl AggregateExpr for Sum {
     }
 
     fn create_sliding_accumulator(&self) -> Result<Box<dyn Accumulator>> {
-        Ok(Box::new(SumAccumulator::try_new(&self.data_type)?))
+        Ok(Box::new(SlidingSumAccumulator::try_new(&self.data_type)?))
```

Comment:
This follows the same pattern as  `MinMaxSlidingAccumulator` https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-expr/src/aggregate/min_max.rs#L664